apps/fyler/src/fyler_server.erl

```diff
@@ -423,16 +426,20 @@ send_to_manager(#task{category = Category} = Task, State) ->
 %% Send response to task initiator as HTTP Post with params <code>status = success|failed</code> and <code>path</code - path to download file if success.
```

Comment:
Line 426 is too long: %% Send response to task initiator as HTTP Post with params <code>status = success|failed</code> and <code>path</code - path to download file if success..